test/torchaudio_unittest/autograd_cuda_test.py

```diff
@@ -0,0 +1,12 @@
+import torch
+from torchaudio_unittest.common_utils import (
+    PytorchTestCase,
+    skipIfNoCuda,
+)
+from torchaudio_unittest.autograd_test_impl import AutogradTestCase
+
+
+@skipIfNoCuda
+class AutogradCUDATest(AutogradTestCase, PytorchTestCase):
```

Comment:
I think it would be nice to have a device generic class similar to what we have in PyTorch for autograd tests https://github.com/pytorch/pytorch/blob/a3a2150409472fe6fa66f3abe9e795303786252c/test/test_autograd.py#L7931-L7937

Reply:
Sure, but let's differ on changing the test infrastructure. This is the common pattern used in torchaudio, so if we are going to change, I would like to change them in one go.